internal/reporters/readme.go

```diff
@@ -32,14 +33,18 @@ func (cr *readmeReporter) SpecDidComplete(specSummary *types.SpecSummary) {
 		return
 	}
 
-	path := ""
+	tablePath := ""
+	resultPath := ""
 	contents := map[string][]string{}
+	header := ""
 	contentKeys := []string{}
-
+  
 	for key, value := range specSummary.Measurements {
-		if path == "" {
-			dirName := getSubDirectory(value.Name, cr.ReportDir)
-			path = filepath.Join(dirName, "README.md")
+		if resultPath == "" {
+			nameComponents := strings.Split(value.Name, " - ")
+			header = sanitize.BaseName(nameComponents[len(nameComponents)-1])
+			tablePath = filepath.Join(cr.ReportDir, "table.md")
+			resultPath = filepath.Join(cr.ReportDir, "result.md")
```

Comment:
This can be initialized at the top. And the table of contents path should just be the readme path.

Reply:
agree